apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -358,6 +360,13 @@ def is_port_promiscuous(port):
                 details['endpoint_group_name'])
         details.update(
             self.get_vrf_details(context, vrf_id=network['tenant_id']))
+        try:
+            self.apic_manager.vmm_shared_secred
```

Comment:
That's a guard for checking that the feature is enabled in apicapi. Could be moved in the attestator.